plugin/ochttp/example_test.go

```diff
@@ -26,3 +27,17 @@ func ExampleTransport() {
 	}
 	_ = client // use client to perform requests
 }
+
+var usersHandler http.Handler
+
+func ExampleHandler() {
+	// Enables OpenCensus for the default serve mux.
+	http.Handle("/users", usersHandler)
+	log.Fatal(http.ListenAndServe("localhost:8080", &ochttp.Handler{}))
+}
+
+func ExampleHandler_mux() {
+	mux := http.NewServeMux()
+	mux.Handle("/users", usersHandler)
+	log.Fatal(http.ListenAndServe("localhost:8080", &ochttp.Handler{Handler: mux}))
```

Comment:
Could you add a Propagation here to exhibit that modification?

Reply:
Done